From 502609207849397fe4064e5709b15b221996fa10 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <vladimir.oltean@nxp.com>
Date: Wed, 15 Aug 2018 14:36:46 +0300
Subject: [PATCH] Makefile: do not install init scripts

* This prevents the SJA1105 switch from being initialized on boot.
  This way, its RGMII ports will remain tristated and will not interfere
  electrically with the 1588 output coming from the LS1021.

Signed-off-by: Vladimir Oltean <vladimir.oltean@nxp.com>
---
 Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index 49159ad..04bfc26 100644
--- a/Makefile
+++ b/Makefile
@@ -130,8 +130,6 @@ install-binaries: $(SJA1105_LIB) $(SJA1105_BIN)
 	install -m 0755 -D etc/etsec_mdio $(DESTDIR)${bindir}/etsec_mdio
 
 install-configs: etc/sja1105-init etc/sja1105.conf
-	install -m 0755 -D etc/sja1105-link-speed-fixup $(DESTDIR)${sysconfdir}/init.d/S46sja1105-link-speed-fixup
-	install -m 0755 -D etc/sja1105-init $(DESTDIR)${sysconfdir}/init.d/S45sja1105
 	install -m 0644 -D etc/sja1105.conf $(DESTDIR)${sysconfdir}/sja1105/sja1105.conf
 
 install-manpages: $(MANPAGES)
-- 
2.7.4

